bundles/tools.vitruv.change.correspondence/src/tools/vitruv/change/correspondence/view/CorrespondenceModelViewFactory.java

```diff
@@ -0,0 +1,21 @@
+package tools.vitruv.change.correspondence.view;
+
+import tools.vitruv.change.correspondence.Correspondence;
+import tools.vitruv.change.correspondence.CorrespondenceFactory;
+import tools.vitruv.change.correspondence.model.CorrespondenceModel;
+
+public final class CorrespondenceModelViewFactory {
+	public CorrespondenceModelViewFactory() {
+		
+	}
+	
+	public static <C extends Correspondence> CorrespondenceModelView<Correspondence> createCorrespondenceModelView(CorrespondenceModel correspondenceModel) {
```

Comment:
```suggestion
	public static CorrespondenceModelView<Correspondence> createCorrespondenceModelView(CorrespondenceModel correspondenceModel) {
```
Do we need the type parameter here?

Reply:
The parameter does not seem to be necessary. Potentially, it is some leftover from trying different ways of typing the views.